docs/CHANGELOG.asciidoc

```diff
@@ -60,6 +61,329 @@ Review important information about the {kib} 8.x releases.
+* Updates Indices API to support sourcemap parameters ({kibana-pull}177847[#177847]).
+* Fixes sorting instances table results on server-side ({kibana-pull}174164[#174164]).
+Cases::
+* Adds support default values for required custom fields ({kibana-pull}175961[#175961]).
+* Collects custom fields telemetry metrics for cases plugin in {kib} ({kibana-pull}175425[#175425]).
+* Persists all filter options of the cases table in the URL ({kibana-pull}175237[#175237]).
+Dashboard::
+* Adds step setting for range slider control ({kibana-pull}174717[#174717]).
+* Adds a deprecation badge in Dashboard on legacy control panels ({kibana-pull}174302[#174302]).
+* Adds external link icon ({kibana-pull}174407[#174407]).
+* Re-adds filtering settings in the Control settings UI ({kibana-pull}172857[#172857]).
+* Adds numeric options list ({kibana-pull}172106[#172106]).
+Discover::
+* Hides the Empty fields section if there are no fields in it ({kibana-pull}172956[#172956]).
+* Adds tabs to Discover and Log Explorer ({kibana-pull}171467[#171467]).
+* Adds multiple improvements to saved query management ({kibana-pull}170599[#170599]).
+* Adds header row height configuration ({kibana-pull}175501[#175501]).
+* Adds new fields ingested in the background with valid mappings ({kibana-pull}172329[#172329]).
+* Adds caching to data view field list request with a `stale-while-revalidate` strategy ({kibana-pull}168910[#168910]).
+* Distinguish among empty and available fields in Discover {esql} mode ({kibana-pull}174585[#174585]).
+Elastic Security::
+For the Elastic Security 8.13.0 release information, refer to {security-guide}/release-notes.html[_Elastic Security Solution Release Notes_].
+Elastic Search::
+* Adds `ignore_empty_value` to generated set processor ({kibana-pull}175172[#175172]).
+Fleet::
+* Adds `skipRateLimitCheck` flag to the Upgrade API and Bulk_upgrade API ({kibana-pull}176923[#176923]).
+* Adds making datastream rollover lazy ({kibana-pull}176565[#176565]).
+* Stops creating  the `{type}-{datastet}@custom` component template during package installation ({kibana-pull}175469[#175469]).
+* Adds the `xpack.fleet.isAirGapped` flag ({kibana-pull}174214[#174214]).
+* Adds a warning when download upgrade is failing ({kibana-pull}173844[#173844]).
+* Adds a message explaining why an agent is not upgradeable ({kibana-pull}173253[#173253]).
+* Makes logs-* and metrics-* data views available across all spaces ({kibana-pull}172991[#172991]).
+* Adds flag for pre-release to templates/inputs endpoint ({kibana-pull}174471[#174471]).
+* Adds concurrency control to Fleet data stream API handler ({kibana-pull}174087[#174087]).
+* Adds a handlebar helper to percent encode a given string ({kibana-pull}173119[#173119]).
+Integrations::
+* Enables minute frequency for incremental syncs ({kibana-pull}176603[#176603]).
+Lens & Visualizations::
+* Adds clickable badges on compact view for {esql} queries ({kibana-pull}176568[#176568]).
```

Comment:
This should be moved to a new ES|QL section as it is about the eS|QL editor and not specific to Visualizations